pihole.8

```diff
@@ -56,269 +58,273 @@ Available commands and options:
 .br
-	(version options):
+    (version options):
 .br
-		-c, --current   Return the current version
+      -c, --current     Return the current version
 .br
-		-l, --latest    Return the latest version
+      -l, --latest      Return the latest version
 .br
-		--hash          Return the Github hash from your local repositories
+      --hash            Return the Github hash from your local repositories
 .br
 
 \fBuninstall\fR
 .br
-	Uninstall Pi-hole from your system
+    Uninstall Pi-hole from your system
 .br
 
 \fBstatus\fR
 .br
-	Display the running status of Pi-hole subsystems
+    Display the running status of Pi-hole subsystems
 .br
 
 \fBenable\fR
 .br
-	Enable Pi-hole subsystems
+    Enable Pi-hole subsystems
 .br
 
 \fBdisable\fR	[time]
 .br
-	Disable Pi-hole subsystems, optionally for a set duration
+    Disable Pi-hole subsystems, optionally for a set duration
 .br
 		
 .br
-	(time options):	
+    (time options):	
 .br
-		#s  Disable Pi-hole functionality for # second(s)
+      #s                Disable Pi-hole functionality for # second(s)
 .br
-		#m  Disable Pi-hole functionality for # minute(s)
+      #m                Disable Pi-hole functionality for # minute(s)
 .br
 
 \fBrestartdns\fR
 .br
-	Restart Pi-hole subsystems
+    Restart Pi-hole subsystems
 .br
 
 \fBcheckout\fR [repo][branch]	
 .br
-	Switch Pi-hole subsystems to a different Github branch
+    Switch Pi-hole subsystems to a different Github branch
 .br
 
-	(repo options):
+    (repo options):
 .br
-		core        Change the branch of Pi-hole's core subsystem
+      core              Change the branch of Pi-hole's core subsystem
 .br
-		web         Change the branch of Admin Console subsystem
+      web               Change the branch of Admin Console subsystem
 .br
-		ftl         Change the branch of Pi-hole's FTL subsystem
+      ftl               Change the branch of Pi-hole's FTL subsystem
 .br
-	(branch options):
+    (branch options):
 .br
-		master      Update subsystems to the latest stable release
+      master            Update subsystems to the latest stable release
 .br
-		dev         Update subsystems to the latest development release
+      dev               Update subsystems to the latest development release
+.br
+      branchname        Update subsystems to the specified branchname
 .br
-
 .SH "EXAMPLE"
+
 Some usage examples
 .br
 
-	Whitelist/blacklist manipulation
+    Whitelist/blacklist manipulation
 .br
 
-	\fBpihole -w iloveads.example.com\fR	Add "iloveads.example.com" to whitelist
+    \fBpihole -w iloveads.example.com\fR  Add "iloveads.example.com" to whitelist
 .br
-	\fBpihole -b -d noads.example.com\fR	Remove "noads.example.com" from blacklist 
+    \fBpihole -b -d noads.example.com\fR  Remove "noads.example.com" from blacklist 
 .br
-	\fBpihole -wild ads.example\fR		Add "ads.example" as wildcard - would block ads.example.net
+    \fBpihole -wild example\fR            Add "example" as wildcard - would block ads.example.net, example.com etc.
```

Comment:
The wildcard block only blocks subdomains, so `example` would not block `example.com`, but `example.com` would block `test.example.com`.